circe/src/test/scala/io/finch/circe/test/package.scala

```diff
@@ -0,0 +1,16 @@
+package io.finch.circe
+
+import cats.Comonad
+import com.twitter.util.Try
+
+package object test {
+
+  implicit val comonadEither: Comonad[Try] = new Comonad[Try] {
```

Comment:
Can we defined this within `AbstractJsonSpec`? It seems to be a place where it's used.